css/style.css

```diff
@@ -16,4 +15,178 @@ body {
+
+/* 1.0 HEADER PART */
+
+/* 1.1 making space between the contents in header  */
+header {
+  display: flex;
+  justify-content: space-around;
+  height: 100px;
+}
+
+/* 1.2 setting height for Karma logo */
+#logo {
+  height: 75px;
+}
+
+/* 1.3 making the list in header (Meet Karma, How it Works, etc) all appear in row */
+li {
+  display: inline-block;
+  margin: 5px;
+  padding: 5px; /* is it better to use padding or margin??? */
+}
+
+/* 1.4 adding weight to the 'Meet karma' font */
+#meetKarma {
+  font-weight: 600;
+}
+
+/* 1.5 decorating the text inside the anchor elements */
+a {
+  text-decoration: none;
+  color: rgb(92, 92, 92);
+}
+
+/* 1.6 adding the 'Hoover' state to the links (NOTE: this will be applied to all the links on the page)  */
+a:hover {
+  color: rgb(255, 86, 24);
+}
+
+/* 2.0 INTRO SECTION TO KARMA */
+
+/* 2.1 adding background-image, decorating font size for texts, and displaying the contents as a column  */
+.intro {
+  height: 750px; /* is there another method to fit the img inside the section without setting height */
+  background-image: url("/home/coder/Documents/GitHub/HTML-CSS-Coursework-Week2/img/first-background.jpg");
+  background-repeat: no-repeat;
+
+  font-size: 70px;
+  font-weight: 100;
+  line-height: 0px;
+  color: aliceblue;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+/* 2.2 making the text "Bring WiFi with you, everywhere you go." appear smaller than what was specified above in 2.1 */
+#firstText {
+  font-size: 40px;
+  font-weight: 100;
+}
+
+/* 2.3 decorating the 'Learn More' button */
+#learnMore {
+  background-color: rgb(255, 86, 24);
+  padding: 35px;
+  margin-bottom: 100px;
+  margin-top: 50px;
+  font-size: 30px;
+  font-weight: 400;
+  border-style: hidden;
+  border-radius: 5px;
+}
+
+/* 3.0 decorating the intro text to the second part of the page (e.g How Karma Works)   */
+.part-one {
+  font-size: 70px;
+  font-weight: 100;
+  color: rgb(46, 46, 46);
+  text-align: center;
+  padding-top: 100px;
+  padding-bottom: 50px;
+}
+
+/* 4.0 HOW KARMA WORKS */
+
+/* 4.1 decorating the child elements inside this section */
+.contents {
+  display: flex;
+  justify-content: space-evenly;
+  font-size: 40px;
+  font-weight: 100;
+  color: rgb(46, 46, 46);
+}
+
+/* 4.2 adding content height, adjusting the display and aligning the texts */
+#content-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 400px;
+  text-align: center;
+}
+
+/* 4.3 adding content height, adjusting the display and aligning the texts */
+#content-2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 400px;
+  text-align: center;
+}
+
+/* 4.4 adding content height, adjusting the display and aligning the texts */
+#content-3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 400px;
+  text-align: center;
+}
```

Comment:
Avoid using IDs in CSS at all costs! They are not re-usable and are too hard to overwrite. 

You should use CSS classes, and re-use them. Here, you have 3 different IDs containing exactly the same styles. Create just one CSS class, and re-use it 3 times in your HTML.